certifier_service/ccaverify/ccaverify.go

```diff
@@ -0,0 +1,63 @@
+//  Copyright (c) 2023, VMware Inc, and the Certifier Authors.  All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ccaverify
+
+/*
+#cgo CFLAGS: -g -Wall -I ../ccalib
+#cgo LDFLAGS: -L ../ccalib -lccaverify -Wl,-rpath=./ccalib:../../certifier_service/ccalib:../../../certifier_service/ccalib
```

Comment:
@yelvmw -- I tried to also include `-rpath` to specify the dir-path to locate `third_party/islet/lib/libislet_sdk.so`, but that caused a regression when `simple_app` is run.

Starting the Certifier Service ends up wanting to locate this `libislet_sdk.so` even for non-CCA-based apps.

This, then, creates a dependency that Islet SDK -must- be installed for all apps, not just for CCA-apps.

So, I decided to remove that `--rpath` specification here.
Instead, now at run-time, users will have to `export LD_LIBRARY_PATH=$CERTIFIER_PROTOTYPE/third_party/islet/lib` ...

run_example.sh has been fixed to adopt this style.